# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1484853193 18000
#      Thu Jan 19 14:13:13 2017 -0500
# Node ID 19b6a417e5e56f552c80cde92c87a0b1c479d0c1
# Parent  e02f1da280efb84a617d59613a7e091b1d6773a2
* pt.h (meets_condition): Delete extraneous function declaration.

diff --git a/libinterp/parse-tree/pt.h b/libinterp/parse-tree/pt.h
--- a/libinterp/parse-tree/pt.h
+++ b/libinterp/parse-tree/pt.h
@@ -27,17 +27,16 @@ along with Octave; see the file COPYING.
 
 #include <string>
 
 #include <iosfwd>
 
 class octave_function;
 class tree_walker;
 class bp_table;
-bool meets_condition (std::string *);
 
 // Base class for the parse tree.
 
 class
 tree
 {
 public:
 
